x/participationrewards/keeper/callbacks.go

```diff
@@ -47,7 +61,12 @@ func (c Callbacks) RegisterCallbacks() icqtypes.QueryCallbacks {
 	a := c.
 		AddCallback("validatorselectionrewards", Callback(ValidatorSelectionRewardsCallback)).
 		AddCallback("osmosispoolupdate", Callback(OsmosisPoolUpdateCallback)).
-		AddCallback("epochblock", Callback(SetEpochBlockCallback))
```

Comment:
It looks like you have created consts for the existing callbacks, but not used them above?